less/variables.less

```diff
@@ -1 +1,7 @@
-// Variables
\ No newline at end of file
+// Variables
+@footer-color: #ebe6e6;
+@font-color: #2b3247;
+@nav-color: #f5f0f0; 
+@brdr-color:#dbd4d4;
```

Comment:
Great job naming your variables! I know what each is for just by looking at the name.